component/admin/language/en-GB.com_jevents.ini

```diff
@@ -1587,4 +1587,7 @@ JEV_ACL_OF_WHO_CAN_SET_PRIORITY="Who can set the priority?"
 JEV_ACL_OF_WHO_CAN_SET_PRIORITY_TIP="This controls what ACL Type can set the priority"
 JEV_ACL_OF_WHO_CAN_SET_PRIORITY_0="Global Event Publishers"
 JEV_ACL_OF_WHO_CAN_SET_PRIORITY_1="Event Creators"
-JEV_ACL_OF_WHO_CAN_SET_PRIORITY_2="Event Editors"
\ No newline at end of file
+JEV_ACL_OF_WHO_CAN_SET_PRIORITY_2="Event Editors"
+
+JEV_LIST_OF_EVENTS_SET_ROW_BGCOLOR="Set event background color?"
```

Comment:
Need to make clear this is a frontend feature

Also needs to tie in to com_calForceCatColorEventForm option on event editing tab

Reply:
Try 'Set event background colour in event list views?"